### PR TITLE
[shelly] Support for Shelly Vintage

### PR DESCRIPTION
**Pull request by markus7017**
This PR adds support for Shelly Vintage, similar to Duo, but no color temp

### CODE REVIEW

File: bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/coap/ShellyCoapHandler.java [hunk 1]
```diff
@@ -606,6 +576,52 @@ private void handleStatusUpdate(String devId, String payload, int serial) {
...
+            }
+
+            if ((profile.settings.relays != null) && (r >= 0) && (r < profile.settings.relays.size())) {
+                ShellySettingsRelay relay = profile.settings.relays.get(r);
+                logger.trace("{}: Coap update for button (type {})", thingName, relay.btnType);
+                if ((s.value != 0) && (relay.btnType.equalsIgnoreCase(SHELLY_BTNT_MOMENTARY)
+                        || relay.btnType.equalsIgnoreCase(SHELLY_BTNT_MOM_ON_RELEASE)
+                        || relay.btnType.equalsIgnoreCase(SHELLY_BTNT_DETACHED))) {
+                    String trigger = "";
+                    switch (new Double(s.value).intValue()) {
```
[the remark this answers is not in the document]
Comment (markus7017) [reply bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/coap/ShellyCoapHandler.java:604]: changed

File: bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/coap/ShellyCoapHandler.java [hunk 2]
```diff
@@ -606,6 +576,52 @@ private void handleStatusUpdate(String devId, String payload, int serial) {
         lastPayload = payload;
     }
 
+    private void handleInput(CoIotDescrSen sen, CoIotSensor s, String rGroup, Map<String, State> updates) {
+        final ShellyDeviceProfile profile = thingHandler.getProfile();
+        int idx = getSensorNumber("Input", sen.id);
+        if (idx > 0) {
```
[the remark this answers is not in the document]
Comment (markus7017) [reply bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/coap/ShellyCoapHandler.java:582]: changed

File: bundles/org.openhab.binding.shelly/src/main/resources/ESH-INF/thing/lights.xml [hunk 3]
```diff
@@ -61,41 +74,41 @@
...
-			<channel id="power" typeId="system.power"/>
-			<channel id="mode" typeId="colorMode"/>
-			<channel id="autoOn" typeId="timerAutoOn"/>
-			<channel id="autoOff" typeId="timerAutoOff"/>
-			<channel id="timerActive" typeId="timerActive"/>
+			<channel id="power" typeId="system.power" />
+			<channel id="mode" typeId="colorMode" />
+			<channel id="autoOn" typeId="timerAutoOn" />
+			<channel id="autoOff" typeId="timerAutoOff" />
+			<channel id="timerActive" typeId="timerActive" />
```
[the remark this answers is not in the document]
Comment (markus7017) [reply bundles/org.openhab.binding.shelly/src/main/resources/ESH-INF/thing/lights.xml:92]: I'm not sure, I always press Shift-Cmd-F, but I noticed that this added some spaces now even I didn't changed something

**Comment by markus7017** [comment]
I added bugfix 7616. The PR is now ready to merge

**Comment by cpmeister** [comment]
My requested changes aren't there...

**Comment by markus7017** [comment]
I added bugfix 7616. The PR is now ready to merge

**Comment by cpmeister** [comment]
Looks like you somehow reverted the changes I requested...

**Comment by markus7017** [comment]
@cpmeister strange, I re-checkedin the changes

**Comment by cpmeister** [comment]
huh, now there are conflicts

**Pull request closed without merging**